whois-commons/src/main/java/net/ripe/db/whois/common/rpsl/DummifierNrtm.java

```diff
@@ -30,10 +30,9 @@ public class DummifierNrtm implements Dummifier {
             AttributeType.ZONE_C
     );
 
-    static final List<AttributeType> ATTRIBUTES_TO_KEEP = Lists.newArrayList(
-            AttributeType.ABUSE_C,
-            AttributeType.LAST_MODIFIED,
-            AttributeType.CREATED);
+    static final Map<ObjectType, List<AttributeType>> ATTRIBUTES_TO_KEEP_OBJECT_TYPE = Map.of(
```

Comment:
abuse-c is only an attribute on organisation, aut-num, inetnum, inet6num. Not on MNTNER ?

Reply:
Do we need to keep attributes by object type?

Reply:
We can keep country for *all* object types and just add it to ATTRIBUTES_TO_KEEP ?

Reply:
If we add country to the current Attributes_to_keep it would be a bit confusing because mntner doesn't have country. So I suggest delete ABUSE_C from MNTNER and keep the map

Reply:
So basically just for ORG and MNTNER we strip all non mandatory attributes so thats why for these two types we need a list to keep optional attributes

Reply:
Regarding ABUSE_C It was added in old code so did not removed. We need to refactor the dummifier class after release and revisit it . So I just did minimal changes

Reply:
we want to keep created and last-modified attributes on *all* object types, so maybe change back to a `List` and allow country on all object types also (including resources)

Reply:
But this function is only for ORG and MNTNER, for rest of objects we do not strip any optional attributes

Reply:
OK! I see! The  field and function naming confused me. Then keep the Map, but remove ABUSE_C from MNTNER.

Reply:
Looks good thanks!